inc/Project.php

```diff
@@ -421,12 +422,12 @@ public function set_text_domain( string $name ): bool {
 	 *
 	 * @since 3.0.0
 	 *
-	 * @return null|string Last updated time if stored, null otherwise.
+	 * @return null|DateTime Last updated time if stored, null otherwise.
 	 */
-	public function get_last_updated_time(): ?string {
+	public function get_last_updated_time(): ?DateTime {
 		$time = gp_get_meta( 'project', $this->project->id, static::UPDATE_TIME_KEY );
 
-		return $time ?: null;
+		return $time ? new DateTime( $time ) : null;
```

Comment:
Does this need to set the `DateTimeZone` to UTC by default? Probably for existing metas.